### ISSUE TITLE
Availability of a .Gov Top Level Domain list sorted by government branch

### ISSUE THREAD

**Issue by JamesC101** [GSA/data#129]
Good day. 

Requesting, if possible, GSA (or any organization) that has the .Gov list categorized by government branch, to include Executive, Judicial, Legislative, Other government etc.

Thank you!

**Comment by h-m-f-t** [comment]
Yes! We'll be splitting the `Federal Agency` domain type into executive, judicial, and legislative in the coming weeks.

**Comment by JamesC101** [comment]
Excellent. How may I get access to this?

**Comment by h-m-f-t** [comment]
Watch this space-- literally. Click the `Watch` button in our repository (top-right corner) and you'll be notified of every change we make.

**Comment by konklone** [comment]
@JamesC101 If for some reason you need to split the .gov list by branch earlier than that, here is some code we (GSA) currently use in a production website to split the data by branch, based on some heuristics we've identified:

https://github.com/18F/pulse/blob/master/data/processing.py#L1151-L1175
